mbedtls/tests/support/net.rs

```diff
@@ -17,12 +19,47 @@ pub fn create_tcp_pair() -> IoResult<(TcpStream, TcpStream)> {
         // most socket operations should work the same way, and UnixSocket
         // is too new to be used
         if libc::socketpair(libc::AF_UNIX, libc::SOCK_STREAM, 0, fds.as_mut_ptr()) == 0 {
-            Ok((
-                TcpStream::from_raw_fd(fds[0]),
-                TcpStream::from_raw_fd(fds[1]),
-            ))
+            Ok((TcpStream::from_raw_fd(fds[0]), TcpStream::from_raw_fd(fds[1])))
         } else {
             Err(IoError::last_os_error())
         }
     }
 }
+
+pub fn create_tcp_stream_pair_loopback() -> (TcpStream, TcpStream) {
```

Comment:
Why? What's wrong with `create_tcp_pair`?

Reply:
Emm, it's my old code where I tried to put some latency and package loss on `loopback` interface to test.
will clean them up